<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Курсовая работа. Главная</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>961</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QGroupBox" name="Connect_groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>921</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Соединение</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <widget class="QLabel" name="Connection_status_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>Соединение не установлено</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="Do_connect_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Открыть порт</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Parameters_connect_button">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры соединения</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Do_connection_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Установить соединение</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="Connect_status_frame_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Статус соединения</string>
      </property>
     </widget>
     <widget class="QLabel" name="System_message_label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Системные сообщения</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="System_messages_tableWidget">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>461</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Send_file_gropubox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>921</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Отправить файл</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="Send_file_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Отправить файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="File_name_choosed">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>761</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Choose_file_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Выберите файл:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Open_file_button">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Открыть...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из программы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="To_file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_menu"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Режим работы</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="To_file_menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
  </widget>
  <action name="about_menu">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Получить...</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Отправка...</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Приём...</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Параметры соединения...</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
